docs/faq/tandem_phases.ipynb

```diff
@@ -0,0 +1,289 @@
+    "\n",
+    "phase0.add_state('x', fix_initial=True, fix_final=False)\n",
+    "\n",
+    "phase0.add_state('y', fix_initial=True, fix_final=False)\n",
+    "\n",
+    "phase0.add_state('v', fix_initial=True, fix_final=False)\n",
+    "\n",
+    "phase0.add_control('theta', continuity=True, rate_continuity=True,\n",
+    "                   units='deg', lower=0.01, upper=179.9)\n",
+    "\n",
+    "phase0.add_parameter('g', units='m/s**2', val=9.80665)\n",
+    "\n",
+    "phase0.add_boundary_constraint('x', loc='final', equals=10)\n",
+    "phase0.add_boundary_constraint('y', loc='final', equals=5)\n",
+    "\n",
+    "# Add alternative timeseries output to provide control inputs for the next phase\n",
+    "phase0.add_timeseries('timeseries2', transcription=tx1, subset='control_input')\n",
+    "\n",
+    "#\n",
+    "# Second Phase: Integration of ArcLength\n",
+    "#\n",
+    "phase1 = dm.Phase(ode_class=BrachistochroneArclengthODE, transcription=tx1)\n",
+    "\n",
+    "p.model.add_subsystem('phase1', phase1)\n",
+    "\n",
+    "phase1.set_time_options(fix_initial=True, input_duration=True)\n",
+    "\n",
+    "phase1.add_state('S', fix_initial=True, fix_final=False,\n",
+    "                 rate_source='Sdot', units='m')\n",
+    "\n",
+    "phase1.add_control('theta', opt=False, units='deg', targets='theta')\n",
+    "phase1.add_control('v', opt=False, units='m/s', targets='v')\n",
+    "\n",
+    "#\n",
+    "# Connect the two phases\n",
+    "#\n",
+    "p.model.connect('phase0.t_duration', 'phase1.t_duration')\n",
+    "\n",
+    "p.model.connect('phase0.timeseries2.controls:theta', 'phase1.controls:theta')\n",
+    "p.model.connect('phase0.timeseries2.states:v', 'phase1.controls:v')\n",
+    "\n",
+    "# Minimize arclength at the end of the second phase\n",
+    "phase1.add_objective('S', loc='final', ref=1)\n",
+    "\n",
+    "p.model.linear_solver = om.DirectSolver()\n",
+    "p.setup(check=True)\n",
+    "\n",
+    "p['phase0.t_initial'] = 0.0\n",
+    "p['phase0.t_duration'] = 2.0\n",
+    "\n",
+    "p.set_val('phase0.states:x', phase0.interp('x', ys=[0, 10]))\n",
+    "p.set_val('phase0.states:y', phase0.interp('y', ys=[10, 5]))\n",
+    "p.set_val('phase0.states:v', phase0.interp('v', ys=[0, 9.9]))\n",
+    "p.set_val('phase0.controls:theta', phase0.interp('theta', ys=[5, 100]))\n",
+    "\n",
+    "p['phase0.parameters:g'] = 9.80665\n",
+    "\n",
+    "p['phase1.states:S'] = 0.0\n",
+    "\n",
+    "dm.run_problem(p)\n",
+    "\n",
+    "fig, (ax0, ax1) = plt.subplots(2, 1)\n",
+    "fig.tight_layout()\n",
+    "ax0.plot(p.get_val('phase0.timeseries.states:x'), p.get_val('phase0.timeseries.states:y'), '.')\n",
+    "ax0.set_xlabel('x (m)')\n",
+    "ax0.set_ylabel('y (m)')\n",
+    "ax1.plot(p.get_val('phase1.timeseries.time'), p.get_val('phase1.timeseries.states:S'), '+')\n",
+    "ax1.set_xlabel('t (s)')\n",
+    "ax1.set_ylabel('S (m)')\n",
+    "plt.show()"
```

Comment:
The plot isn't showing up on this one.

Reply:
Try this again now that I've merged in the latest